### DISCUSSION TITLE
How to Enable BigQuad Features in Latest Crazyflie Firmware?

### DISCUSSION THREAD

**Discussion by Amin-Yazdanshenas** [bitcraze/discussions#1010] [Q&A] [answered]
Hello everyone,

I am working with a Crazyflie 2.X and a BigQuad deck, and I am trying to build the firmware to enable the BigQuad features. I understand that the build system has transitioned to Kbuild, and the previous method of using **config.mk** to set build defines has been replaced.

I've gone through the [Bitcraze documentation on customizing the firmware with Kbuild](https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/development/kbuild/), and I've attempted to use make menuconfig to customize the build. However, I am unsure of the exact steps to enable BigQuad features within this new system. Previously, it was done by defining the ENABLE_BQ_DECK flag.

Could someone please guide me on the exact steps or the equivalent setting in the new Kbuild system to enable BigQuad features? Additionally, are there any new flags or configurations specific to the BigQuad deck that I should be aware of in the latest firmware versions?

Thank you in advance for your assistance!

**Comment by krichardsson** [comment, marked as answer]
Hi,

1. Enter the configuration GUI by typing `make menuconfig` in a terminal
2. Go to the `Expansion deck configuration  --->` menu
3. Check the `Support the BigQuad deck` option
4. Now some more options related to the BigQuad deck are displayed, choose the ones that are appropriate for your setup but the default settings are probably fine
5. Pres <ESC> a few times until you get the `Do you wish to save your new configuration?` popup, save and you are done

**Comment by krichardsson** [comment]
Btw, you might want to run `make cf2_defconfig` before the steps above to get a good configuration base to start from.

**Comment by Amin-Yazdanshenas** [comment]
Tnx that worked for me
